conda/plugins/manager.py

```diff
@@ -554,6 +595,22 @@ def get_environment_specifiers(self, filename: str) -> CondaEnvironmentSpecifier
             name=filename, plugin_names=[hook.name for hook in hooks]
         )
 
+    def get_environment_specifier(self, filename: str, plugin_name=None) -> CondaEnvironmentSpecifier:
```

Comment:
Is it worth is to have this function? Can callers of the plugin manager just call `detect_environment_spec_plugin_from_file` or `get_explicit_environment_specifier` directly?